core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -1508,6 +1508,54 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     servers.foreach(assertNoExemptRequestMetric(_))
   }
 
+  @Test
+  def testRateMetricsHaveCumulativeCount() {
+    val numRecords = 1000
```

Comment:
Would 100 records be enough for this test?

Reply:
yes, updated.